Bottom: 292c7f88d1f85c2e5c73f00973c4c7f293b8bbcf
Top:    e1869f35665bbd7f5551e3e7bf53cdf7437d9671
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 20:54:51 +0100

Refresh of hid-gtk-remove-usage-of-deprec

---

diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 3be7931..7136fbb 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -479,7 +479,6 @@ ghid_range_control (GtkWidget * box, GtkWidget ** scale_res,
   gtk_scale_set_value_pos (GTK_SCALE (scale), pos);
   gtk_scale_set_draw_value (GTK_SCALE (scale), set_draw_value);
   gtk_scale_set_digits (GTK_SCALE (scale), digits);
-  gtk_range_set_update_policy (GTK_RANGE (scale), GTK_UPDATE_CONTINUOUS);
 
   /* Increments don't make sense, use -1,1 because that does closest to
      |  what I want: scroll down decrements slider value.
